### ISSUE TITLE
UHP: bestmove without options?

### ISSUE THREAD

**Issue by paulmelis** [jonthysell/Mzinga#114]
Seems there's a third (undocumented) form of `bestmove`, i.e. without options, but it doesn't seem to return a result anytime soon?

**Comment by jonthysell** [comment]
Yes. While not part of the UHP protocol, calling bestmove by itself will just run the search until you send a cancel (Ctrl+C).

**Comment by paulmelis** [comment]
Okay, then maybe the Ctrl+C doesn't work on Linux. As I tried that earlier, but it simply keeps computing.
